go-controller/pkg/ovn/policy.go

```diff
@@ -605,13 +597,20 @@ func (oc *Controller) createMulticastAllowPolicy(ns string, nsInfo *namespaceInf
 		}
 	}
 
+	// for the update case need to delete the old style of multicast port group
```

Comment:
I was going to say we should open an issue to remind us to eventually remove this code, but you know what? No one is actually using multicast support yet. Let's just not bother with cleanup...

Reply:
done